### ISSUE TITLE
feat: implement `hideTags` on `sendImage` & `sendFile`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1463]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.9.2](https://github.com/open-wa/wa-automate-nodejs/compare/3.9.1...3.9.2) (2021-03-12)

- build(deps): bump swagger-ui-dist from 3.44.1 to 3.45.0 [`#1466`](https://github.com/open-wa/wa-automate-nodejs/pull/1466)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.16.1 to 4.17.0 [`#1457`](https://github.com/open-wa/wa-automate-nodejs/pull/1457)
- build(deps-dev): bump @typescript-eslint/parser from 4.16.1 to 4.17.0 [`#1459`](https://github.com/open-wa/wa-automate-nodejs/pull/1459)
- build(deps): bump patch-package from 6.4.5 to 6.4.6 [`#1458`](https://github.com/open-wa/wa-automate-nodejs/pull/1458)
- build(deps-dev): bump @types/node from 14.14.32 to 14.14.33 [`#1461`](https://github.com/open-wa/wa-automate-nodejs/pull/1461)
- build(deps): bump uuid from 3.4.0 to 8.3.2 [`#1451`](https://github.com/open-wa/wa-automate-nodejs/pull/1451)
- build(deps-dev): bump @types/marked from 1.2.2 to 2.0.0 [`#1452`](https://github.com/open-wa/wa-automate-nodejs/pull/1452)
- 🐛  update loop fix #1465 [`#1465`](https://github.com/open-wa/wa-automate-nodejs/issues/1465)
- 📝  docs update [`d71664c`](https://github.com/open-wa/wa-automate-nodejs/commit/d71664c198f4060915b8b398ccff5544cee01737)
- Release 3.9.2 [`6e6b6b2`](https://github.com/open-wa/wa-automate-nodejs/commit/6e6b6b213384af01a0eed86a4447701e80d2069b)
- ✨ feat: hideTags` on `sendImage` & `sendFile` #1463 [`92e4a4e`](https://github.com/open-wa/wa-automate-nodejs/commit/92e4a4e102614b4512fb4586d61709db9b9692f0)
- 🥚  #1464 [`6e4b576`](https://github.com/open-wa/wa-automate-nodejs/commit/6e4b576faf326f657b18b3911cff9b20308251c4)
- 🥚  patch update [`19db6eb`](https://github.com/open-wa/wa-automate-nodejs/commit/19db6eb161cb5149423d25e84141991c62dccd24)
- 🥚  patch update [`436afb2`](https://github.com/open-wa/wa-automate-nodejs/commit/436afb20d7e3dabea289707de7fb8820e621120e)
- added emoji support for CI [`9976170`](https://github.com/open-wa/wa-automate-nodejs/commit/9976170b0d20b2f1fb2880e2fe0816cca54aa853)
- 🥚  #1454 [`a4baf72`](https://github.com/open-wa/wa-automate-nodejs/commit/a4baf728aeb8a2088be62e3ff3645b73435be57b)
- updated release-image [`7ec481f`](https://github.com/open-wa/wa-automate-nodejs/commit/7ec481f89fe5058c41d82b808f1c9abe8fabf921)
